tests/e2e/common-operator-integ-suite.sh

```diff
@@ -87,6 +108,8 @@ initialize_variables() {
   COMMAND="kubectl"
   ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
   KUBECONFIG="${KUBECONFIG:-"${ARTIFACTS}/config"}"
+  LOCALBIN="${LOCALBIN:-"${HOME}/.local/bin"}"
```

Comment:
In [Makefile.core.mk](https://github.com/istio-ecosystem/sail-operator/blob/c5485e96934a9873fe57844f558d847f980aca29/Makefile.core.mk#L432) file, we use operator-sdk from the $PWD/bin, can we re-use the same path (as defaults)?